modules/test/playwright/tests/journal-web/journalArticleAutosave.spec.ts

```diff
@@ -35,7 +36,37 @@ const autoSaveTest = mergeTests(
 	}),
 	isolatedSiteTest,
 	journalPagesTest,
-	loginTest()
+	loginTest(),
+	pagesAdminPagesTest
+);
+
+autoSaveTest(
+	'LPD-36442 UndoRedo Should not appear when editing default values',
```

Comment:
Hey @NemethNorbert I just realised about this, could you move the LPD to the `@tag` naming?